src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = wall.querySelector('.spider');
+const wallSize = wall.offsetWidth;
+const spiderSize = spider.offsetWidth;
```

Comment:
For this task, it's ok (when spider and wall are square), but if elements will have different width and height you should calculate them too